scripts/scil_tractogram_assign_uniform_color.py

```diff
@@ -134,6 +147,20 @@ def main():
             for key in dict_colors.keys():
                 if key in base:
                     color = dict_colors[key]
+
+            if color is None and args.overwrite:
+                updated_dict = True
+                logging.warning('No color found for {} in your '
+                                'dict_colors. Generating a new color.'
+                                .format(base))
+                # Generate a new color that is not in the dict_colors
+                # and not already used.
+                new_color = distinguishable_colormap(nb_colors=1,
+                                                     exclude=rgb_colors)
```

Comment:
You should append this new color to rgb_colors?
If you have to add 5-10 new colors it would be important to not repick them?